x/posts/internal/types/models/post_response_test.go

```diff
@@ -167,21 +167,19 @@ func TestPostQueryResponse_String(t *testing.T) {
 		models.NewUserAnswers(
 			models.NewUserAnswer([]models.AnswerID{models.AnswerID(1)}, liker),
 		),
-		[]models.ReactionQueryResponse{
-			models.NewReactionQueryResponse("https://example.com/like", ":like:", liker),
-			models.NewReactionQueryResponse("👍", ":+1:", otherLiker),
+		[]models.PostReaction{
+			models.NewPostReaction("https://example.com/like", ":like:", liker),
+			models.NewPostReaction("👍", ":+1:", otherLiker),
 		},
 		models.PostIDs{
 			"dd065b70feb810a8c6f535cf670fe6e3534085221fa964ed2660ebca93f910d1",
 			"dd065b70feb810a8c6f535cf670fe6e3534085221fa964ed2660ebca93f910d1",
 		},
 	)
 
-	expected := `
-ID: dd065b70feb810a8c6f535cf670fe6e3534085221fa964ed2660ebca93f910d1
-Reactions: [{https://example.com/like :like: cosmos1s3nh6tafl4amaxkke9kdejhp09lk93g9ev39r4} {👍 :+1: cosmos15lt0mflt6j9a9auj7yl3p20xec4xvljge0zhae}]
-Children: [dd065b70feb810a8c6f535cf670fe6e3534085221fa964ed2660ebca93f910d1, dd065b70feb810a8c6f535cf670fe6e3534085221fa964ed2660ebca93f910d1]
-`
+	expected := `ID: dd065b70feb810a8c6f535cf670fe6e3534085221fa964ed2660ebca93f910d1
+Reactions: [{"owner":"cosmos1s3nh6tafl4amaxkke9kdejhp09lk93g9ev39r4","shortcode":"https://example.com/like","value":":like:"} {"owner":"cosmos15lt0mflt6j9a9auj7yl3p20xec4xvljge0zhae","shortcode":"👍","value":":+1:"}]
+Children: [dd065b70feb810a8c6f535cf670fe6e3534085221fa964ed2660ebca93f910d1, dd065b70feb810a8c6f535cf670fe6e3534085221fa964ed2660ebca93f910d1]`
```

Comment:
Please revert these lines as they should not change

Reply:
everything should be fine now, there was a problem inside the `PostReaction` type where its fields are alfabetically ordered (probably due to a refactor that I've done).

Reply:
There is still something fishy about the tests. You are creating the reactions as follows: 

```golang
[]models.PostReaction{
  models.NewPostReaction(":like:", "https://example.com/like", liker),
  models.NewPostReaction(":+1:", "👍", otherLiker),
},
```

But then you are testing if the result is 

```json
[
  {
    "owner": "cosmos1s3nh6tafl4amaxkke9kdejhp09lk93g9ev39r4",
    "shortcode": "https://example.com/like",
    "value": ":like:"
  }
  {
    "owner": "cosmos15lt0mflt6j9a9auj7yl3p20xec4xvljge0zhae",
    "shortcode": "👍",
    "value": ":+1:"
  }
]
```

As you can see in the result the `value` and `shortcode` values are inverted. There is either an error in the test or an error somewhere else. Please make the test expected result correct (invert those values) and fix the bug accordingly.

Reply:
yeah I've still need to commit the changes